scripts/list_dataset_metadata.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple, List
```

Comment:
I've created a new file for list_dataset_metadata command because in my opinion `core.py` should only contain a set of entrypoints that just delegate the call to the command controllers.

Reply:
agreed.